testsuite/features/reposync/srv_wait_for_reposync.feature

```diff
@@ -1,8 +1,9 @@
-# Copyright (c) 2019 SUSE LLC
+# Copyright (c) 2019-2021 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature: Abort all reposync activity
+Feature: Wait for reposync activity to finish
 
+@regular_ci
```

Comment:
Maybe the word "ci_tests" instead? or just "ci"?

Reply:
Maybe `continuous_integration then

We could also avoid using new tags, and just use a different feature file. But it seemed to me more consistent that way.